includes/manifest.php

```diff
@@ -0,0 +1,194 @@
+<?php
+/**
+ * HM Platform Plugin Manifest.
+ */
+
+namespace HM\Platform;
+
+// Cavalcade.
+Plugin::register( 'cavalcade', 'plugins/cavalcade/plugin.php' )
```

Comment:
Can we look at how this would work in a typical object resgitration pattern?
```
$registration = Plugin::register( 'cavalcade' ... );
$registration->set_data( $data );
```

This would be much more consistent with WP norms.

Reply:
You could write it like that with the current codebase I guess but I'm looking at ways to flatten this all down to a big ol' array